actor/xkcd/README.md

```diff
@@ -0,0 +1,51 @@
+# Xkcd comic generator
+
+This example actor demonstrates three capability providers:
+- wasmcloud:httpserver  for responding to http requests
+- wasmcloud:httpclient  for fetching http resources from the internet
+- wasmcloud:numberge for generating a random number.
```

Comment:
This should be `wasmcloud:builtin:numbergen`

Reply:
fixed